cmake/features.cmake

```diff
@@ -10,6 +10,8 @@ ie_dependent_option (ENABLE_INTEL_CPU "CPU plugin for OpenVINO Runtime" ON "RISC
 
 ie_dependent_option (ENABLE_ARM_COMPUTE_CMAKE "Enable ARM Compute build via cmake" OFF "ENABLE_INTEL_CPU" OFF)
 
+ie_dependent_option (ENABLE_LLMDNN "Enable library for LLM in CPU plugin, need intel compiler 2023.0+" OFF "ENABLE_INTEL_CPU" OFF)
```

Comment:
does this library work for ARM platforms?

Reply:
No, it needs AMX instruction and only supports SPR+.

Reply:
Then, we need to add a condition for this option

Reply:
Condition added: x86_64 and Linux.